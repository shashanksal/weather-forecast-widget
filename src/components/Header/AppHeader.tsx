import React from 'react';

import {
    AppBar,
    Typography,
    Toolbar,
    Paper
} from '@material-ui/core';

const styles = {
    headStyle: {
        textAlign: 'center',
        display: 'flex',
        alignItems: 'center',
        justifyContent: 'center',
    }
}

const AppHeader = (props: any) => {
    return (
        <Paper>
            <AppBar position="static" style={{ background: '#009688' }}>
                <Toolbar>
                    <Typography variant="h5" className="headStyle" align="center" display="block">
                        Weather Forecast Tool
                </Typography>
                </Toolbar>
            </AppBar>
        </Paper>
    )
}

export default AppHeader;
