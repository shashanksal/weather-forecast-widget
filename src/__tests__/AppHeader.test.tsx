import React from "react";
import { render, fireEvent, waitForElement } from "@testing-library/react";

import AppHeader from '../components/Header';

describe('test category', () => {
	it("AppHeader matches snapshot" , () =>{
		const {asFragment} = render(<AppHeader />);
		expect(asFragment()).toMatchSnapshot();
	})
})
