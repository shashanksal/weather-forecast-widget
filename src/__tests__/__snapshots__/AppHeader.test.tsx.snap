// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test category AppHeader matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
      style="background: rgb(0, 150, 136); align-items: center;"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter"
        >
          Weather Forecast Tool
        </h5>
      </div>
    </header>
  </div>
</DocumentFragment>
`;
